Bowling/Bowling/Frame.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BowlingApp
+{
+    public class Frame
+    {
+        public int frameNumber { get; set; }
+        public int firstRoll { get; set; }
+        public int secondRoll { get; set; }
+        public int bonusRoll { get; set; }
+        public string frameOutcome { get; set; }
+        public int frameScore { get; set; }  
```

Comment:
I decided to go ahead and keep framescore in this class because it would be possible for framescore to be different based on a strike or spare rolls in upcoming frames.